<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartPower</class>
 <widget class="QMainWindow" name="SmartPower">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Smart Power</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_startstop">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="deviceConnectedStatus">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Device Not Detected: Verify Connection/Correct Firmware</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_valtage">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>98</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltage</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_voltage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_ampere">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_ampere">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>187</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ampere</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_watt">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_watt">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>307</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Watt</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_wh">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>451</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_wh">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>428</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Watt/hour</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_onoff">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>On</string>
    </property>
   </widget>
   <widget class="QwtPlot" name="qwtPlot">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>440</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_sw_version">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>n/a</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>S/W Version :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_fw_version">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>n/a</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>F/W Version :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_a_max_static">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>267</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>41</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>41</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Max : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_a_min_static">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>267</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>41</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>41</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Min : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_w_max_static">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>387</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Max : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_w_min_static">
    <property name="geometry">
     <rect>
      <x>134</x>
      <y>389</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_a_max">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>259</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_a_min">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>259</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_w_max">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>381</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_w_min">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>381</y>
      <width>66</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plot_change">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>483</x>
      <y>50</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Watt Graph</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_log">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>97</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <zorder>pushButton_onoff</zorder>
   <zorder>pushButton_startstop</zorder>
   <zorder>label_3</zorder>
   <zorder>deviceConnectedStatus</zorder>
   <zorder>label_valtage</zorder>
   <zorder>lcdNumber_voltage</zorder>
   <zorder>lcdNumber_ampere</zorder>
   <zorder>label_ampere</zorder>
   <zorder>lcdNumber_watt</zorder>
   <zorder>label_watt</zorder>
   <zorder>lcdNumber_wh</zorder>
   <zorder>label_wh</zorder>
   <zorder>qwtPlot</zorder>
   <zorder>label_sw_version</zorder>
   <zorder>label_4</zorder>
   <zorder>label_fw_version</zorder>
   <zorder>label_5</zorder>
   <zorder>label_a_max_static</zorder>
   <zorder>label_a_min_static</zorder>
   <zorder>label_w_max_static</zorder>
   <zorder>label_w_min_static</zorder>
   <zorder>label_a_max</zorder>
   <zorder>label_a_min</zorder>
   <zorder>label_w_max</zorder>
   <zorder>label_w_min</zorder>
   <zorder>pushButton_plot_change</zorder>
   <zorder>checkBox_log</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
